packages/wallets/src/features/accounts/screens/PoiPoaDocsSubmitted/PoiPoaDocsSubmitted.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { useEffect } from 'react';
+import { useInvalidateQuery } from '@deriv/api-v2';
+import { DerivLightWaitingPoiIcon } from '@deriv/quill-icons';
+import { WalletButton, WalletsActionScreen } from '../../../../components';
+import { useModal } from '../../../../components/ModalProvider';
+import './PoiPoaDocsSubmitted.scss';
+
+const PoiPoaDocsSubmitted = () => {
+    const invalidate = useInvalidateQuery();
+    const { hide } = useModal();
+
+    // need invalidate queries in order to update status badge of CFD account
+    useEffect(() => {
+        return () => {
+            invalidate('get_account_status');
+            invalidate('mt5_login_list');
```

Comment:
hey @wojciech-deriv 
here we are using invalidate because after we close the resubmission modal, we are showing our TH main page with MT5 accounts. But since these status badges are rendered based on `mt5_acc_list` and `get_acc_status` data, after closing the modal we should somehow invalidate these data in order to change the badge status on the main screen. hope it was clear

Reply:
as invalidate only marks data as no longer valid, without triggering the actual update
the actual update will be fired on the main page because under the hood, the hook which is responsible for status badge use `mt5_acc_list` and `get_acc_status` hooks, that eventually will trigger the rerender